lib/core/lib/ops/userOps.js

```diff
@@ -513,6 +551,43 @@ function buildUserOps (db/*, config */) {
       })
     },
 
+    /**
+     * List the teams to which the user belongs to
+     * Does not go recursively to parent teams
+     *
+     * @param  {Object}   params { id, organizationId, limit, page }
+     * @param  {Function} cb
+     */
+    listUserTeams: function listUserTeams ({ id, organizationId, page = 1, limit }, cb) {
+      Joi.validate({ id, organizationId, page, limit }, validationRules.listUserTeams, function (err) {
+        if (err) return cb(Boom.badRequest(err))
+
+        const pageLimit = limit || config.get('authorization.defaultPageSize')
+        const offset = (page - 1) * pageLimit
+
+        const job = {
+          id,
+          organizationId,
+          offset,
+          limit: pageLimit,
+          user: {},
+          client: db
+        }
+
+        readUserTeams(job, (err) => {
+          if (err) return cb(err)
+          const pageSize = Math.min(pageLimit, job.user.teams ? job.user.teams.length : 0)
+          const result = {
```

Comment:
In addition to the previous comment these naming are kind of confusing. Considering that
```
  page: Joi.number().integer().min(1).description('Page number, starts from 1'),
  limit: Joi.number().integer().min(1).description('Items per page'),
```
what do you think about: `currentPage` instead of `page`, `pageSize` instead of `limit` (and here you always return page size not the length of the current returned data)?
This is optional ofc, but the limit value must be fixed

Reply:
the `limit` naming seems confusing also to me but that change is a breaking change and should be done on all endpoints at once